resources/icons/16px/bp3-data-connection-16.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
the name of this file does not match the icon name. it must be `data-connection.svg`, as shown in the [build failure](https://app.circleci.com/pipelines/github/palantir/blueprint/376/workflows/7bcbd439-021d-4f93-b20e-aae13fd5ae47/jobs/35040):

```
lerna ERR! yarn run compile stderr:
(node:248) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/home/circleci/project/resources/icons/16px/data-connection.svg'
```